### PR TITLE
Check for doctrine migration

### PR DESCRIPTION
**Pull request by glaubinix**
New check to make sure all migrations are applied if you use doctrine migrations.
I had to use mocks for the tests because otherwise a database connection is required.

Note: the doctrine/migrations library does not really maintain tags this is why I used dev-master as dev dependency.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -24,7 +24,8 @@
         "guzzle/http"                : "3.*",
         "guzzle/plugin-mock"         : "3.*",
         "videlalvaro/php-amqplib"    : "2.*",
-        "predis/predis"              : "0.8.*"
+        "predis/predis"              : "0.8.*",
+        "doctrine/migrations"        : "dev-master"
```
Comment (lsmith77) [inline composer.json]: better use ``1.0@dev``

**Comment by lsmith77** [comment]
other than that minor comment looks good to me.

@Thinkscape ?

**Comment by glaubinix** [comment]
@lsmith77 pushed.

And seems like the tests are failing on master too due to the same segfault.

**Comment by lsmith77** [comment]
hmm .. I somehow seem to remember having seen similar failures in a different repository as well.

**Comment by kbond** [comment]
Looks to be the same segfault as #52 - we need to figure out what test is causing it.

**Comment by lsmith77** [comment]
ah damn. so it wasnt a different repo. can one of you adjust the travis.yaml to run phpunit in verbose mode so that we can see which test is causig this?

**Comment by glaubinix** [comment]
@lsmith77 pushed the verbose flag. I also skipped 2 tests which were in the area where I saw the segfaults. But there is something really weird, different php versions fail on different tests.

**Comment by lsmith77** [comment]
argh sorry .. we also need ``--debug``

**Comment by Thinkscape** [comment]
@glaubinix would be so kind and fix indentations ?

**Comment by glaubinix** [comment]
@Thinkscape are you talking about the 2 lines where I marked the tests as skipped. This PR should not be merged like this. I first need to remove all the debugging again. But I am not sure where we are at with the failing build.

**Comment by Thinkscape** [comment]
Ah, this https://travis-ci.org/zendframework/ZendDiagnostics/jobs/59508302#L335 ?

**Comment by glaubinix** [comment]
@Thinkscape yes, but different PHP Versions throw the segfault for different tests.

**Comment by till** [comment]
Tests seem to run locally, only difference is XDebug. Could that cause it?

**Comment by lsmith77** [comment]
likely ..

**Comment by till** [comment]
@lsmith77 Confirmed in #59 — xdebug causes the failures.

**Comment by lsmith77** [comment]
please rebase

**Comment by glaubinix** [comment]
rebase done

**Pull request merged**